### PR TITLE
Feature/2016 refactor kudos Closes #2016

### LINKED ISSUE #2016: Refactor Kudos to Home Page
- Add Kudos to home page
- Only showing last month of Kudos
- Only showing `public` Kudos
- Able to add new Kudos
- Adjust for different screen sizes (design might need help)

### PR DESCRIPTION
**Pull request by jahanner**

### CODE REVIEW

**Comment by ZacharyKlein** [comment]
@mkimberlin Is this PR still valid? Should we try to resolve the merge conflicts and pull this in, or close it?

**Comment by mkimberlin** [comment]
> @mkimberlin Is this PR still valid? Should we try to resolve the merge conflicts and pull this in, or close it?

@ZacharyKlein This needs discussion in standup.

File: server/src/main/java/com/objectcomputing/checkins/services/kudos/Kudos.java [hunk 1]
```diff
@@ -0,0 +1,188 @@
+package com.objectcomputing.checkins.services.kudos;
+
```
Comment (mvolkmann) [inline server/src/main/java/com/objectcomputing/checkins/services/kudos/Kudos.java:2]: test

File: server/src/main/java/com/objectcomputing/checkins/services/kudos/Kudos.java [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+import io.micronaut.core.annotation.Introspected;
```
Comment (mvolkmann) [inline server/src/main/java/com/objectcomputing/checkins/services/kudos/Kudos.java:3]: test

**Pull request closed without merging**